tests/fake_lab/tasks/heating.py

```diff
@@ -18,22 +18,22 @@ def __init__(
     ):
         super().__init__(samples=samples, *args, **kwargs)
         self.setpoints = setpoints
-        self.sample = samples[0]
 
     def run(self):
```

Comment:
docstrings are missing and explanation for the numbers

Reply:
added